### PR TITLE
README reworked

### PR DESCRIPTION
**Pull request by comradekingu**
Weblate mentioned in README, as per
https://hosted.weblate.org/projects/quotes-status-creator/quotes-status-creator/#alerts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,35 +7,38 @@
 </h1>
 
 <p align="center">
-	Quotes Status Creator lets you share quotations as status images on social media
+	Share quotes as status images on social media
```
Comment (VishnuSanal) [inline README.md:10]: I think `Quotes Status Creator lets you share quotations as status images on social media` was better. It's the short description for our app on IzzyOnDroid, PlayStore etc What do you think?

File: README.md [hunk 2]
```diff
@@ -53,16 +56,17 @@ If you enjoy the app, wish to contribute or simply wish to be with like-minded p
 </p>
 
 ## 🎨 Follow [Instagram Page](https://instagram.com/quotes_status_creator)
-Follow the Instagram Page to get your daily dose of inspiration delivered to your feed! If you enjoy the app, you'll enjoy the Instagram Page too! Come follow us & be inspired!
+Follow the Instagram page to get your daily dose of inspiration delivered to your feed. \
```
Comment (VishnuSanal) [inline README.md:59]: Why the carriage return at the end? Please revert if that was a mistake.

File: README.md [hunk 3]
```diff
@@ -148,7 +152,8 @@ Support the project by [starring it](https://github.com/VishnuSanal/Quotes/starg
 </pre>
 
 ## 📬 Feedback and Suggestion
-We appreciate your effort to help us in making this app more awesome. Your feedback is immensely helpful to us and we love to hear from you. Leave your feedback/suggestions [here](https://t.me/QuotesStatusCreator) or feel free to create an [issue](https://github.com/VishnuSanal/Quotes/issues/new/choose)! :)
+Helpful advice makes the app better. Your feedback is immensely helpful, and we love to hear from you. \
```
Comment (VishnuSanal) [inline README.md:155]: Why the carriage return at the end? Please revert if that was a mistake.

**Review by VishnuSanal — CHANGES_REQUESTED** [review]
Thanks for the changes, please see comments.

File: README.md (hunk 1 above)
Comment (comradekingu) [reply README.md:10]: I agree, and here in the README it says "Quotes Status Creator" right above it, so it isn't needed here.

File: README.md (hunk 2 above)
Comment (comradekingu) [reply README.md:59]: It is intentional. That way the sentences both start on a newline.

File: README.md (hunk 3 above)
Comment (comradekingu) [reply README.md:155]: It is intentional. That way the sentences both start on a newline.

Currently it is
Sentence 1 lorem ipsum. Sentence 2 lorem ipsum.

And I want it to be

Sentence 1 lorem ipsum.
Sentence 2 lorem ipsum.

**Review by VishnuSanal — APPROVED** [review]

**Pull request merged**